### ISSUE TITLE
homeassistant.const.HTTP_* removed

### ISSUE THREAD

**Issue by travi** [leikoilja/ha-google-home#415]
## Describe the bug

As [mentioned in the breaking chnages section of the release notes from core v2021.12](https://www.home-assistant.io/blog/2021/12/11/release-202112/#updates-for-custom-integration-developers), homeassistant.const.HTTP_* were removed. It is mentioned that stdlib http.HTTPStatus should be used instead.

## Version of the integration

If you are unsure about the version check the [github release here](https://github.com/leikoilja/ha-google-home/releases).

```text
v1.9.3
```

## Logs

```text
Setup failed for google_home: Unable to import component: cannot import name 'HTTP_NOT_FOUND' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)
```

**Comment by KapJI** [comment]
It was fixed in 1.9.2, please update the integration.

**Comment by travi** [comment]
Apologies. I overlooked that I still had an update pending. Thank you for the quick guidance.
